src/index.html

```diff
@@ -5,9 +5,52 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class ="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Product image"
+        class="card__image"
+      >
+
+      <div class="card__content">
+        <h1 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
I think a `p` tag would be better here. H1-H6 tags should represent section headings. Check [the docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)

Reply:
I thougth as it's the header of the card, if I change to p my whole code goes crap. Is it really that much of a problem ?